docker-entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+
+sleep 5  # waits for another container to be ready
```

Comment:
A little bit more failure resistant solution on docker docs (wait-for-postgres.sh): https://docs.docker.com/compose/startup-order/.